PackagingAndRepos.ipynb

```diff
@@ -332,6 +308,26 @@
     "The ``-u`` is a convenience that means from then on you can use just ``git push`` and ``git pull`` to send your code to and from github."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: If you used the GitHub \"add a README\" or \"add a LICENSE\" option, your repository won't be empty and the above `push` command won't work.  In that case you'll need to pull down the content github auto-created and add yours on top of it.  The commands below will do that.\n",
+    "\n",
+    "**You don't need to do this cell if the above worked**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!git fetch <yourgithubusername> master\n",
+    "!git rebase <yourgithubusername>/master\n",
+    "!git push <yourgithubusername> master -u"
```

Comment:
I don't think "-u" is necessary at this point? At least its absence did not give any error when I told people to `git push username master` after a rebase at the workshop.

Reply:
Although maybe it's safer anyway to do this in case they don't run the other cell? (I'm not sure, though: is there any danger to `-u`)

Reply:
Oh, actually, I am *not* sure. I never use `git pull` or `git push` just like that. I'm ok with leaving `-u` in then and see what happens next year. 😉